cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-cancel.cy.js

```diff
@@ -0,0 +1,137 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import FileManager from '../../../support/utils/fileManager';
+import DateTools from '../../../support/utils/dateTools';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+import UsersSearchPane from '../../../support/fragments/users/usersSearchPane';
+
+let user;
+const invalidUsername = getRandomPostfix();
+const usernamesFileName = `'valid_and_invalid_users_uuids-${getRandomPostfix()}.csv`;
+const nextThreeMonthsDate = DateTools.getAfterThreeMonthsDateObj();
+
+describe('bulk-edit', () => {
+  describe('in app approach', () => {
+    before('Create test data', () => {
+      cy.createTempUser(
+        [
+          Permissions.bulkEditLogsView.gui,
+          Permissions.bulkEditUpdateRecords.gui,
+          Permissions.uiUserEdit.gui,
+        ],
+        'staff',
+      ).then((userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: TopMenu.bulkEditPath,
+          waiter: BulkEditSearchPane.waitLoading,
+        });
+        FileManager.createFile(
+          `cypress/fixtures/${usernamesFileName}`,
+          `${user.username}\r\n${invalidUsername}`,
+        );
+      });
+    });
+
+    after('Delete test data', () => {
+      FileManager.deleteFile(`cypress/fixtures/${usernamesFileName}`);
+      Users.deleteViaApi(user.userId);
+    });
+
+    it(
+      'C397355 Verify CANCEL during Bulk edit Users In app  (firebird) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+      () => {
+        // Select the "Users" radio button on  the "Record types" accordion
+        BulkEditSearchPane.checkUsersRadio();
+        // Select  "Usernames" option from the "Record identifier" dropdown
+        BulkEditSearchPane.selectRecordIdentifier('Usernames');
+        // Verify landing page shows "Drag and drop or choose file with  Usernames" text
+        BulkEditSearchPane.verifyDragNDropUsernamesArea();
```

Comment:
`verifyDragNDropUsernamesArea` already has `checkUsersRadio` and `selectRecordIdentifier('Usernames')` inside